os/mm/mm_heap/mm_getheap.c

```diff
@@ -219,6 +219,11 @@ int mm_get_heapindex(void *mem)
 	if (mem == NULL) {
 		return 0;
 	}
+	if (BASE_HEAP == NULL) {
```

Comment:
Is this possible to be null?

Reply:
If this function is called before application is loaded, then it might result into null.
We have faced this issue during porting so added this condition